packages/deployer/extensions/delete.me

```diff
@@ -0,0 +1 @@
+// Dummy file to include the folder in git
```

Comment:
I wouldn't delete these. I think they're there to keep the folders around in version control.

Reply:
Yes, I added them to get the folders in version control while we doesn't have any script there. As soon as we add something real we should delete the files. Otherwise hardhat will fail when our tasks scan for those folders.

Reply:
For this case, is common practice to leave a `.gitkeep` file, which really is nothing more than something to track